### PR TITLE
SPCTR-2044: Fix - Border: Misaligned reset control button issue.

### PR DESCRIPTION
**Pull request by raojip**
### Description
- https://brainstormforce.atlassian.net/browse/SPCTR-2044
- Fix - Border: Misaligned reset control button issue.

### Screenshots
- Here is the tested video:- https://d.pr/v/rtmfOQ

### Types of changes

Bug fix (non-breaking change which fixes an issue)

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: src/styles/control-header.scss [hunk 1]
```diff
@@ -29,7 +29,6 @@
 
 		.uagb-reset {
 			padding: 0;
-			margin-right: 10px;
```
Comment (RajkiranBagal) [inline src/styles/control-header.scss:32]: This margin right applies to all uag-resets, Did you confirm if all resets in all controls wherever this class is present is working fine?

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: src/styles/control-header.scss (hunk 1 above)
Comment (raojip) [reply src/styles/control-header.scss:32]: Yes, I Tested this with all blocks, it working fine. (Tested video:- https://d.pr/v/mrPeGZ)

Comment (RajkiranBagal) [reply src/styles/control-header.scss:32]: I did not say all Blocks @raojip. Please read the comment carefully.

Comment (RajkiranBagal) [reply src/styles/control-header.scss:32]: I can see in the video you shared for spacing control the reset icon is misaligned.

Comment (raojip) [reply src/styles/control-header.scss:32]: Yes, I removed margin-right: 10px; from uag-resets class, and I checked where we use this class is working fine.
basically, where we need margin-right:10px there I added padding-left or padding-right.

Comment (RajkiranBagal) [reply src/styles/control-header.scss:32]: I see issue in video itself. @AbhijitNage123 Can you please review.

Comment (AbhijitNage123) [reply src/styles/control-header.scss:32]: @raojip See here - https://share.getcloudapp.com/geuO8d60 reset icon is not aligned properly with the preset setting. Verify other settings too. Use the grid ruler extension to check the alignment of the reset icon.

**Review by AbhijitNage123 — CHANGES_REQUESTED** [review]

File: src/styles/control-header.scss (hunk 1 above)
Comment (raojip) [reply src/styles/control-header.scss:32]: Fixed the alignment issue with the preset setting, and tested it properly for all settings, now working fine, please check.

**Review by AbhijitNage123 — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**